ngraph/cmake/external_onnx.cmake

```diff
@@ -57,19 +37,13 @@ macro(onnx_set_target_properties)
     endif()
 endmacro()
 
-FetchContent_GetProperties(ext_onnx)
-if(NOT ext_onnx_POPULATED)
-    FetchContent_Populate(ext_onnx)
-    set(ONNX_USE_PROTOBUF_SHARED_LIBS ${BUILD_SHARED_LIBS} CACHE BOOL "Use dynamic protobuf by ONNX library")
-    set(ONNX_NAMESPACE ${NGRAPH_ONNX_NAMESPACE})
-    set(ONNX_USE_LITE_PROTO ${NGRAPH_USE_PROTOBUF_LITE} CACHE BOOL "Use protobuf lite for ONNX library")
-    set(ONNX_ML ON CACHE BOOL "Use ONNX ML")
-    if(CMAKE_CROSSCOMPILING)
-        set(ONNX_CUSTOM_PROTOC_EXECUTABLE ${SYSTEM_PROTOC})
-    endif()
-
-    add_subdirectory(${ext_onnx_SOURCE_DIR} ${ext_onnx_BINARY_DIR} EXCLUDE_FROM_ALL)
-    onnx_set_target_properties()
-else()
-    onnx_set_target_properties()
+set(ONNX_USE_PROTOBUF_SHARED_LIBS ${BUILD_SHARED_LIBS} CACHE BOOL "Use dynamic protobuf by ONNX library")
+set(ONNX_NAMESPACE ${NGRAPH_ONNX_NAMESPACE})
+set(ONNX_USE_LITE_PROTO ${NGRAPH_USE_PROTOBUF_LITE} CACHE BOOL "Use protobuf lite for ONNX library")
+set(ONNX_ML ON CACHE BOOL "Use ONNX ML")
+if(CMAKE_CROSSCOMPILING)
+    set(ONNX_CUSTOM_PROTOC_EXECUTABLE ${SYSTEM_PROTOC})
 endif()
+
+add_subdirectory(${CMAKE_SOURCE_DIR}/thirdparty/onnx ${CMAKE_BINARY_DIR}/_deps/onnx EXCLUDE_FROM_ALL)
+onnx_set_target_properties()
```

Comment:
can we move content of this script directly to https://github.com/openvinotoolkit/openvino/blob/master/thirdparty/CMakeLists.txt? 

We are going to have a single folder with all 3rdparty 

And later merge ngraph / ie into one `runtime` folder.